src/components/SignUp.js

```diff
@@ -43,21 +44,32 @@ export default class SignUp extends Component {
 								});
 	}
 
-	showNotification = (notification, notificationType, time = 5000) => {
+	redirectToLoginBtn = e => {
+		this.setState({ redirect: true });
+		if(this.state.redirect) {
```

Comment:
if operator here is senseless. this.state.redirect always true

Reply:
done